docs/stock/pointandfigure.md

```diff
@@ -0,0 +1,23 @@
+Point and Figure chart
+================
+
+
+A point-and-figure (P&F) chart is a technical analysis tool used in financial markets to track price movements without too much focus on the time. It uses a series of Xs and Os to represent upward and downward price movements, respectively. The Xs indicate price increases, while the Os indicate price decreases. Each symbol is added to the column only when the `boxSize` threshold is crossed. A new column is created only when there is a reversal in the price trend of a predefined amount, known as the `reversalAmount`. This approach filters out minor price fluctuations, focusing only on significant price changes, which helps identify trends, support, and resistance levels. Point-and-figure charts are particularly useful for spotting breakouts, trends, and potential entry or exit points in the market.
+
+![pointandfigure.png](pointandfigure.png)
+
+
+### Data
+
+Point and Figure series accepts the data only in `[x, close]` format, which then gets calculated internally to `[{ x, [y], marker }]` groups with unique x values for each array element, which creates up and down trend columns. After that, the groups are parsed into Highcharts data format of `[{ x, y, marker }]`, where for each `y` value in the group column, a new point is created, to create individual points in Point and Figure columns.
+
+### New options
+Point and Figure series adds the following features to the chart:
+- new `cross` symbol,
+- `boxSize` API option,
+- `reversalAmount` API option,
+- separate API options for `marker` and `markerUp`
```

Comment:
What do you think about making these options URLs? Personally, I would love to click these options (if avail) instead of searching for them.